src/components/Back.tsx

```diff
@@ -0,0 +1,12 @@
+import back from "../assets/back.svg";
+interface BackProps {
+  handleClick: () => void;
+}
+
+export default function Back({ handleClick }:BackProps) {
```

Comment:
Give a descriptive name to this component. It's hard to recognize which kind it is. Choose a name like, `BackButton` or something similar.